### DISCUSSION TITLE
Exit `Note source`

### DISCUSSION THREAD

**Discussion by newmedicine** [TriliumNext/Trilium#4285] [General]
There doesn't seem to be any way of exiting `Note source` view accessed as below:

<img width="297" alt="image" src="https://github.com/zadam/trilium/assets/22443747/2f5cbded-85db-44b6-8c63-600a0abb412b">

Any thoughts please?

Thanks

Nick

**Comment by zerebos** [comment]
I think `existing` was a typo for `editing` in which case, yes there is no way to edit the html of a text note directly through the GUI. I'm fairly certain that's intentional.

**Reply by newmedicine** [reply]
Thanks, it was more that I can't cancel out of the source view (without clicking elsewhere in the sidebar note tree and then back onto the note title). I'm not expecting to edit.

**Reply by zerebos** [reply]
Ohhhh you said `exiting` not `existing`, my bad 😅 

Yes you're right that does not seem to be possible from my testing either, I think that would make a good bug report/feature request.

**Comment by meichthys** [comment]
@newmedicine What version of trilium are you on?
in the latest stable v0.60.4, the note source is just opened in a new tab, so all you would need to do it close the tab.

**Reply by newmedicine** [reply]
Oops! You're right
